src/api/base.ts

```diff
@@ -12,21 +13,33 @@ export const api = ky.create({
   headers: {},
 });
 
-export const privateApi = (token: string) =>
+const getToken = () => {
+  const cookies = nookies.get();
+
+  return cookies.token;
+};
+
+export const privateApi = () =>
   ky.create({
     prefixUrl: apiUrl,
-    headers: {
-      Authorization: `Bearer ${token}`,
+    hooks: {
+      beforeRequest: [
+        (request) => {
+          const token = getToken();
```

Comment:
getToken is not reused. So this can be 

```suggestion
         const { token } = nookies.get();
```